src/components/core/accordions/offer-accordion.tsx

```diff
@@ -234,22 +234,23 @@ export const OfferAccordionHeader = ({
     string | undefined
   >();
 
-  const tokenOffers = useSelector(
-    (state: RootState) => state.marketplace.tokenOffers,
+  const nftOffers = useSelector(
+    (state: RootState) => state.marketplace.nftOffers,
   );
 
   const topOffer: OffersTableItem = useMemo(
-    () => tokenOffers && tokenOffers[0],
-    [tokenOffers],
+    () => nftOffers && nftOffers[0],

```

Comment:
If testing, should you asset it has a length? Are you sure it always has at least 1 item>?
```suggestion
    () => Array.isArray(nftOffers) && nftOffers.length && nftOffers[0],

```

Reply:
Since we are checking condition `nftOffers[0]` to find if zeroth position is having values. It's working fine. 

But yeah updated with logic you shared Helder. Thanks 😊

Reply:
@prasanthLalapeta that is correct, it wouldn't fail, my bad

Reply:
@prasanthLalapeta it's also less verbose, so maybe best to put it back if you wish

Reply:
Yeah reverted. Thanks @heldrida